.gitleaks.toml

```diff
@@ -10,7 +25,17 @@
 [[ rules ]]
     id = "generic-api-key"
     [ rules.allowlist ]
-        commits = [
-            "474554504e7e33cef2a71774f825d5b3947ff797",
-            
+
+        paths = [
+            # Allowlists a false positive detection at  
+            # https://github.com/apollographql/apollo-ios/blob/474554504e7e33cef2a71774f825d5b3947ff797/Tests/ApolloCodegenTests/TestHelpers/ASTMatchers.swift#L72
+            # This was previously allowlisted via commit hash, but updating that rule
+            # To support allowlisting false positive detections in the files below as well.
+            '''Tests/ApolloCodegenTests/TestHelpers/ASTMatchers.swift''',
+
+            # Allowlist the various high-entropy strings in xcscmblueprint files
+            '''Apollo.xcodeproj/project.xcworkspace/xcshareddata/Apollo.xcscmblueprint$''',
```

Comment:
Similarly, these files shouldn't be a problem once we merge the repo change because they no longer exist.

Reply:
Although we will instead have [`apollo-ios-xcframework`](https://github.com/apollographql/apollo-ios-xcframework) which will still contain an xcode project file.

Reply:
Yea the `xcodeproj` are all gone from everywhere except the `apollo-ios-xcframework` repo @calvincestari  mentioned so it may be the only place this is needed going forward after the repo changes are fully released.